docs/community/topics/i18n.rst

```diff
@@ -65,13 +65,25 @@ These steps cover how to add or change text that has been marked as translateabl
 
       pybabel extract . -F babel.cfg -o src/pydata_sphinx_theme/locale/sphinx.pot -k '_ __ l_ lazy_gettext'
 
+   **To run this in ``.nox``**: ``nox -s translate -- extract``.
+
 #. Update the message catalogs (``PO`` files) with `the PyBabel update command <https://babel.pocoo.org/en/latest/cmdline.html#update>`__:
 
    .. code-block:: bash
 
       pybabel update -i src/pydata_sphinx_theme/locale/sphinx.pot -d src/pydata_sphinx_theme/locale -D sphinx
 
+   **To run this in ``.nox``**: ``nox -s translate -- update``.
+
+
 This will update these files with new information about the position and text of the language you have modified.
+If you change non-translatable text, you can run pybabel and it will update the line numbers.
+However, this step is optional - the line numbers are to inform the human translator, not to perform the translation.
```

Comment:
Making a separate paragraph, to clarify that this step is only optional if all you're doing is editing non-translatable text.

```suggestion

If you *only* change non-translatable text (like HTML markup), the `extract` and `update` commands will only update the positions (line numbers) of the translatable strings. Updating positions is optional - the line numbers are to inform the human translator, not to perform the translation.
```